src/i18n/locales/en/translation.json

```diff
@@ -188,6 +188,10 @@
       "help_contact": {
         "part1": "Need help? Contact",
         "part2": "Alby Support"
+      },
+      "unlock_error": {
+        "help": "Your account data is encrypted with your unlock password. If you really forgot your unlock password you have to reset and add your Lightning account again.",
```

Comment:
Small change:

"Your account data is encrypted with your unlock password. If you really forgot your unlock password you have to reset **your data** and add your Lightning account again."

Reply:
wasn't sure if we then need to describe what data? 
and if we say "account data" what does that mean?

Reply:
Yes, true. Might sound scary.